StepsToCare/CareDashboard.ascx.cs

```diff
@@ -1966,6 +1979,11 @@ private void BindMainGrid( RockContext rockContext, CareNeedService careNeedServ
                 qry = qry.Where( b => b.DateEntered <= endDate );
             }
 
+            if ( !cbIncludeFutureNeeds.Checked )
+            {
+                qry = qry.Where( b => b.DateEntered <= futureDate );
```

Comment:
Update date range picker to restrict end date to today.

Reply:
Added end date to datepicker in this commit: https://github.com/KingdomFirst/RockBlocks/pull/114/commits/965f2ccf4d6bd32001336fd09f7ca7acbd51bdb1